runtime/hsa-runtime/core/driver/xdna/amd_xdna_driver.cpp

```diff
@@ -60,27 +61,27 @@ namespace AMD {
 XdnaDriver::XdnaDriver(std::string devnode_name)
     : core::Driver(core::DriverType::XDNA, devnode_name) {}
 
-XdnaDriver::~XdnaDriver() { FreeDeviceHeap(); }
+XdnaDriver::~XdnaDriver() {}
```

Comment:
If it's empty, you don't have to define it (the compiler will auto generate it). Or you can `= default` it.

Reply:
Fixed